### PR TITLE
Improve startup time

### PR DESCRIPTION
**Pull request by trask**

### CODE REVIEW

File: agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/LoggingConfigurator.java [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+import java.nio.file.Path;
+import java.util.Locale;
+
+import static org.slf4j.Logger.ROOT_LOGGER_NAME;
+
+class LoggingConfigurator {
+
+    private final LoggerContext loggerContext;
+
+    public Level level;
```
Comment (heyams) [inline agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/LoggingConfigurator.java:34]: make instance variables private.

File: agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/LoggingConfigurator.java [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+        // Diagnostics IPA log file location. Disabled by default.
+        String diagnosticsOutputDirectory = System.getenv(DiagnosticsHelper.APPLICATIONINSIGHTS_DIAGNOSTICS_OUTPUT_DIRECTORY);
+        if (diagnosticsOutputDirectory != null && !diagnosticsOutputDirectory.isEmpty()) {
+            rootLogger.addAppender(configureDiagnosticAppender(diagnosticsOutputDirectory));
+        }
+
+        if (DiagnosticsHelper.isOsWindows()) {
+            rootLogger.addAppender(configureEtwAppender());
+        }
+        // TODO what about linux?
```
Comment (heyams) [inline agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/LoggingConfigurator.java:81]: a good question.  can we create a work item to track this? i thought app services have something already.. will need to follow up with Jenny.

**Review by heyams — APPROVED** [review]
Added a couple of comments. 👍

File: agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/LoggingConfigurator.java (hunk 2 above)
Comment (trask) [reply agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/LoggingConfigurator.java:81]: I think this is already tracked here: https://msazure.visualstudio.com/One/_workitems/edit/9606582

**Pull request merged**